### ISSUE TITLE
Signed release

### ISSUE THREAD

**Issue by janmojzis** [seznam/goenvtemplator#29]
Hello,
can You please gpg-sign next goenvtemplator release:
~~~
gpg --armor --output "${VER}.tar.gz.asc" --detach-sig "${VER}.tar.gz"
~~~
and attach the "asc" file to the  release.

This helps in creating distribution packages (e.g. Debian),
automated building scripts can then check that the tarball has not been modified in any way.

**Comment by FUSAKLA** [comment]
Hi, I'll take a look at it, we use goreleaser for building the assets but IIRC it should support signing them

**Comment by FUSAKLA** [comment]
Yep, this should be possible https://goreleaser.com/customization/sign/

Actually looking at the goreleaser docs, it supports event the Debian packaging :thinking: 
https://goreleaser.com/customization/nfpm/

**Comment by FUSAKLA** [comment]
Does it have to be a gpg signed? 
It would be way easier using [cosign](https://github.com/sigstore/cosign) in combination with the GitHub OIDC workload identity
with no need to take care of any additional secrets
